<ui version="4.0" >
 <class>GeoSysSettingsIconisedVolumesForm</class>
 <widget class="QWidget" name="GeoSysSettingsIconisedVolumesForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Iconised volumes (click to restore):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_iconisedvolumes" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="_4" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
